index.html

```diff
@@ -163,9 +165,12 @@ <h2 class="contact-me forsize">Contact me</h2>
             If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it 
         </p>
         <form action="https://formspree.io/f/mjvzkgvn" method="POST" class="form">
-            <input type="text" name="user_name" placeholder="Your Name Here" maxlength="30" required>
-            <input type="email" name="user_email" placeholder="Your Email Here" required>
-            <textarea name="user_comment" class="comment" maxlength="500" required placeholder="Write your message here."></textarea>
+            <label for="user_name" hidden>Your Name Here</label>
+            <input type="text" id="user_name" name="user_name" placeholder="Your Name Here" maxlength="30" required>
+            <label for="user_email" hidden>Your Email Here</label>
+            <input type="email" id="user_email" name="user_email" placeholder="Your Email Here" required>
+            <label for="comment" hidden>Write your message here</label>
+            <textarea name="comment" id="comment" maxlength="500" required placeholder="Write your message here."></textarea>
             <button type="submit" class="getresume gettouch">
                 Get it touch
```

Comment:
- [ ] Same for `Get my resume` and `Get in touch` buttons, Please fix the contrast ratio between text color and background color in normal and hover button interactions to enhance the site accessibility.

![image](https://user-images.githubusercontent.com/68796232/187468301-6e74bfb9-da09-4b9c-8055-03dcc5bd5ee2.png)

![image](https://user-images.githubusercontent.com/68796232/187468330-be837d92-0e1d-451e-8175-66f1d9a9711a.png)